COMMUNITY.md

```diff
@@ -0,0 +1,40 @@
+# Engaging in the radare2 community
+
```

Comment:
i would add a paragraph in here saying that we don't like to impose behaviour rules to anyone because we assume that people will behave like well educated adults, or even feel attacked if we dont clearly specify that we have nothing against them, but seems like trusting humans is never a good idea.

Reply:
> i would add a paragraph in here saying that we don't like to impose behaviour rules to anyone because we assume that people will behave like well educated adults, or even feel attacked if we dont clearly specify that we have nothing against them, but seems like trusting humans is never a good idea.

@trufae  @condret  : Maybe it is good to add it under "Rule enforcement" section. (BTW please delete the question mark, thank you)
Like this:

_"We don't like to impose behavior rules to anyone because we assume that each member will behave as a well-mannered and educated adult, but in any multi-cultural community accident may happen. Accordingly we urge that if any member feels annoyed by someone's behavior, we would like the related member to contact us immediately so we can clarify the situation, and to solve the problem._

Reply:
@trufae @condret 

I had my review as per asked, thanks for asking.
The decision is yours, Please decide on what you think is the best for community.

Reply:
Tbh, I'd prefere if people at first try to fix conflicts themself. Otherwise we end up taking a role of an authority, that we/I don't want to be. I'm here to develope r2, not act as someone who has more rights than others, which will inevitably happen if we/i put ourself on that podest.

Reply:
people should address personal conflicts in private and not in public chats with other unrelated people.